rest/data/commit_queue.go

```diff
@@ -38,6 +46,268 @@ func (pc *DBCommitQueueConnector) GetGitHubPR(ctx context.Context, owner, repo s
 	return pr, nil
 }
 
+func (pc *DBCommitQueueConnector) AddPatchForPr(ctx context.Context, projectRef model.ProjectRef, prNum int, modules []restModel.APIModule, messageOverride string) (string, error) {
```

Comment:
95% of the changes in this file are copied from https://github.com/evergreen-ci/evergreen/pull/4408/files#diff-9a641a84e0528bf75d9c6823fc9c565b953f96f209706568a80d7914b8532073R253. The reason the code has not been deleted from the job is to preserve backwards compatibility during the deploy